server/schedulers/hot_region_config.go

```diff
@@ -60,6 +61,7 @@ type hotRegionSchedulerConfig struct {
 	// step = max current * rank step ratio
 	ByteRateRankStepRatio float64 `json:"byte-rate-rank-step-ratio"`
 	KeyRateRankStepRatio  float64 `json:"key-rate-rank-step-ratio"`
+	QPSRankStepRatio      float64 `json:"qps-rank-step-ratio"`
```

Comment:
Is it necessary? It seems we never change these configurations.

Reply:
only unify with the previous configuration

Reply:
I also have doubt if the previous configurations are necessary.